Added clarification on version to download for Umbraco Forms theme files. (#5330)
umbraco/UmbracoDocs: 10/umbraco-forms/developer/themes.md

# Themes

Umbraco Forms supports Themes, allowing forms to be customized in a much simpler manner.

## Creating a Theme

To create a theme, you need to create a folder at `/Views/Partials/Forms/Themes/`. The name of the folder is the name of theme that will be visible in the backoffice when choosing it.

Copy the explicit files you wish to override in your theme, it may be a single file or all files from the `default` theme folder. Make the necessary changes you desire to CSS class names, markup etc.

### Obtaining the Default Theme Files

For Umbraco 9 and previous, it's straightforward to copy the files you need from the default theme folder. We highly recommend that you never customize any files found in the `default` themes folder. There is a risk that any customizations to these files will be lost with any future upgrades you do to Umbraco Forms.

Umbraco 10+ distributes these files as part of a Razor class library, so you won't find them on disk. Instead you should download the appropriate zip file for your Forms version and extract the ones you need.

> You can obtain the latest versions of the default theme for each Forms major version from the following links:

Author: Given there's only one link for this version, probably best as:

```suggestion
You can obtain the latest version of the default theme Forms 10 from the following link:
```

* [Forms 10 Default Theme](https://umbra.co/umbraco-forms-default-theme) (for 10.5.0.rc1 and above)

If you are using a lower minor version of Forms than those listed, you should download an older version of the default theme.

The default theme for minor versions are available from the "Source Code" tab available at the bottom of [Umbraco Forms Package page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available.

You should use the theme available for the highest version that's less than or equal to than the version of Forms you have installed.

> For example, you might be using Umbraco Forms 10.4. No file for that version is available so you should use the one for 10.3.

Reviewer: ```suggestion
For example, when using Umbraco Forms 10.4, and no file for that version is available use version 10.3 instead.
```

Author: I think this one is OK as is (I had already changed it due to the "sentence too long" warning).

Reviewer: Thank you for confirming! I believe in this case it is best to have the 2 this content together so it is more understandable that it is related to the example with 10.4. 

So I will just merge this one "For example, when using Umbraco Forms 10.4, and no file for that version is available use version 10.3 instead." 👍

> For example, if you were using Umbraco Forms 10.4, and no file for that version is available, you would use the one for 10.3.

Author: Resolving as outdated.
